src/Core/CoreKernel.php

```diff
@@ -311,14 +313,17 @@ protected function detectLegacyEnvironment()
     }
 
     /**
-     * If missing configuration, redirect to install.php
+     * If missing configuration, redirect to install.php if it exists.
```

Comment:
`install.php` isn't a thing anymore?

Reply:
Hmm, looks like a legacy function.. Not sure I understand it, but I suppose it doesn't matter.

Reply:
Well we still have https://github.com/silverstripe/silverstripe-installer-wizard. If we decide to actively break it by removing `install.php` references from core, then we should also mark this module as non-functioning. I don't feel strongly either way, not a blocker for this PR.